### DISCUSSION TITLE
Does SDRTrunk update TG automatically

### DISCUSSION THREAD

**Discussion by millertbrad** [DSheirer/sdrtrunk#1636] [Q&A]
When new TGs are added to RR, does it automatically update it on mine or do I have to manually add them? If not, can that feature be added?

**Comment by louisik1** [comment]
No. You have to pull the data again, select what playlist you want to update, and it will. Some folks won't want random updates happening to their playlists and aliases without knowing about it, I think.

**Comment by millertbrad** [comment]
Can you tell me how?

**Comment by louisik1** [comment]
Try the instructions here: https://github.com/DSheirer/sdrtrunk/wiki/Playlist-Editor


Specifically starting with: https://github.com/DSheirer/sdrtrunk/wiki/Playlist-Editor#how-to-login-to-radio-reference

Have you already used this process before?  If you have:

When I want to check for updates, the screen is still set up from the last time I imported. So one thing I do is select a different county I dont want, then select the county I do want, in order to make sdrtrunk get the latest data from radioreference.  So if you see here, the drop down for County is on PSERN.  I will select something else, then go back to PSERN.  The talkgroups in the list below will populate.  
![261876274-d7ee3d61-a240-4a4a-856d-8e855caf0767](https://github.com/DSheirer/sdrtrunk/assets/491335/cd10e72f-fdbb-4e60-98c2-8849a75c3a7a)


You will see a spinner while it is fetching the newest data:
![261875929-a99ad9b9-38bf-4d14-ab61-38a8bd270fe2](https://github.com/DSheirer/sdrtrunk/assets/491335/9bae4ce8-b26d-44af-b17c-6a41ddd3e817)

In this example, I will use my PSERN2 alias list.  When the data is imported, you should see lines where the alias in blank.  These are entries that my  list doesn't have, so they will be new for me to import.  For example, Police Data and Police 1 (Bellevue....) here:
![261875977-3bee58e8-eb59-41ae-97e9-f86ec9916bbb](https://github.com/DSheirer/sdrtrunk/assets/491335/e8468e55-f1f5-48a2-8ff3-ae0536e1489a)

Then click the Import All Talkgroups button
![261875989-4f0b21f0-49a9-4f67-b3f6-8142ad52765f](https://github.com/DSheirer/sdrtrunk/assets/491335/84040288-7484-401d-8d8e-4f12de68b52b)

and you should see the blank ones get filled in, plus any that have changed names will get updated:
![261875998-f0c054ea-6f50-4ca8-a2e5-bf0b211c818c](https://github.com/DSheirer/sdrtrunk/assets/491335/bd10a380-4361-416a-9551-8704d53658e5)

That's it.  Then assign them to your broadcasts or whatever you want to do from there.  I hope this helps.

**Comment by millertbrad** [comment]
So you have to go import all to the alias again?

**Reply by louisik1** [reply]
No, you can just select a single talk group and the user interface on the right gives you options for naming it, and a "create alias" button.

You can create as many alias lists as you want. I encourage you to experiment with it in order to learn how it works.
